featuretools/computational_backends/utils.py

```diff
@@ -278,3 +279,8 @@ def create_client_and_cluster(n_jobs, num_tasks, dask_kwargs, entityset_size):
             warned_of_memory = True
 
     return client, cluster
+
+
+def get_client_cluster():
```

Comment:
maybe just add a comment that you separated this out, to make it easier to mock during testing?

Reply:
Added comment.